Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,453 @@
+    , "SentPackets_Field", sosSentPackets_Field
+    , "SentPackets", sosSentPackets
+    , "ConnectionDuration", sosConnectionDuration
+    , "User", sosUser
+    , "AppRulePolicyId", sosAppRulePolicyId
+    , "AppRulePolicyName", sosAppRulePolicyName
+    , "AppRuleService", sosAppRuleService
+    , "AppRuleType", sosAppRuleType
+    , "AppRuleObject", sosAppRuleObject
+    , "AppRuleObjectContent", sosAppRuleObjectContent
+    , "AppRuleAction", sosAppRuleAction
+    , "SourceIPv6Address", sosSourceIPv6Address
+    , "DestinationIPv6Address", sosDestinationIPv6Address
+    , "ServiceName", sosServiceName
+    , "AppFullString", sosAppFullString
+    , "AppIDNumber", sosAppIDNumber
+    , "AppID", sosAppID
+    , "AppCategoryID", sosAppCategoryID
+    , "AppSignatureID", sosAppSignatureID
+    , "IPSCategoryName", sosIPSCategoryName
+    , "AntiSpywareCategory", sosAntiSpywareCategory
+    , "URLPathName", sosURLPathName
+    , "FileIdentifier", sosFileIdentifier
+    , "DPIInspectedFlow", sosDPIInspectedFlow
+    , "DestinationPortDPT", sosDestinationPortDPT
+    , "NATDestinationPort", sosNATDestinationPort
+    , "SourcePortSPT", sosSourcePortSPT
+    , "NATSourcePort", sosNATSourcePort
+    , "BladeID", sosBladeID
+    , "UUID", sosUUID
+    , "HttpRequestMethod", sosHttpRequestMethod
+    , "FileName", sosFileName
+    , "FileSize", FileSize
+    , "HttpReferrer", sosHttpReferrer
+    , "HttpUserAgent", HttpUserAgent
+    , "CaptureATPVerdict", sosCaptureATPVerdict
+    , "Country", sosCountry
+    , "FirewallAction", sosFirewallAction
+    , "ApplicationID", sosApplicationID
+    , "SourceZone", sosSourceZone
+    , "DestinationZone", sosDestinationZone
+    , "CFSCategoryID", sosCFSCategoryID
+    , "CFSCategoryName", sosCFSCategoryName
+    , "CFSPolicyName", sosCFSPolicyName
+    , "AppControlFileName", sosAppControlFileName
+    , "AppCategoryName", sosAppCategoryName
+    , "AppIDName", sosAppIDName
+    , "AppSignatureName", sosAppSignatureName
+    , "IPSFullString", sosIPSFullString
+    , "IPSSignatureName", sosIPSSignatureName
+    , "LegacyMessageCategory", sosLegacyMessageCategory
+    , "LogMsgCategory", sosLogMsgCategory
+    , "LogMsgNote", sosLogMsgNote
+    , "LogMsgSeverity", sosLogMsgSeverity
+    , "NATSourceIPAddress", sosNATSourceIPAddress
+    , "NATDestinationIPAddress", sosNATDestinationIPAddress
+    , "SourceZoneType", sosSourceZone
+    , "DestinationZoneType", sosDestinationZone
+    , "SourceVPNPolicyName", sosSourceVPNPolicyName
+    , "DestinationVPNPolicyName", sosDestinationVPNPolicyName
+    , "InterfaceStatistics", sosInterfaceStatistics
+    , "SoleraNPCSURL", sosSoleraNPCSURL
+    , "SerialNumber", sosSerialNumber
+    , "DeviceModel", sosDeviceModel
+    , "LogMsgID", sosLogMsgID
+    , "RequestURL", RequestURL
+    , "Activity", EventMessage
+    , "EventMessageDetail", sosEventMessageDetail
+    , "UserSessionType", sosUserSessionType
+    , "UserSessionDuration", sosUserSessionDuration

```

Comment:
@jaimeesc you are adding around 70 key value pairs in AdditionalFields. Are these all relevant for Network data? I hope we are not adding any fields that are non network and will never be populated as part of network logs

Reply:
@jaimeesc  sorry for multiple comments. I just want to make sure we map only relevant network fields and avoid fields that might never be populated.

Reply:
For UserSessionType (DeviceCustomString5Label) and UserSessionDuration (DeviceCustomString6Label), these two specific ones refer to the user that was logged in and associated with the event that was logged. Information included is the username (susr tag parsed from AdditionalExtensions), type, and duration of the session. Can these be included in AdditionalFields, or will I need to exclude them?

Reply:
@jaimeesc 
you can include all fields that are populated in network related logs.